src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -188,9 +189,8 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
         }),
     );
 
-    enrollRequests.pipe(finalize(() => {
-      this.isEnrolling = false;
-    })).subscribe({
```

Comment:
Unrelated change here. The only change that should be observed here is the addition of `this.progressBarService.updateProgress(0);`.